docs/lib/graphqlapi/fragments/ios/advanced-workflows/50_interceptor.md

```diff
@@ -0,0 +1,44 @@
+To include custom headers in your outgoing requests, add an `URLRequestInterceptor` to the `AWSAPIPlugin`. Also specify the name of one of the APIs configured in your **amplifyconfiguration.json** file.
+
+<amplify-block-switcher>
+
+<amplify-block name="Listener (iOS 11+)">
```

Comment:
maybe we don't need the block switcher in this case? It seems like Listener vs Combine doesn't play a role in this particular API.

Reply:
Fair point! Will simplify to a single snippet.